yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -286,9 +286,18 @@ Internet-Draft    Updated YANG Module Revision Handling    December 2022
    revision date, and hence all revisions of a given module or submodule
    MUST have unique revision dates.
 
-   A corollary to the above is that the relationship between two module
-   or submodule revisions cannot be determined by comparing the module
-   or submodule revision date alone, and the revision history, or
+   For a given YANG module revision, revision B is defined as being
+   derived from revision A, if revision A is listed in the revision
+   history for revision B.  Although this document allows for a branched
+   revision history, a given YANG module revision history does not
+   contain all revisions in all possible branches, it only lists those
+   from which is was derived, i.e., the module revision's history
+   describes a single path of derived revisions back to the root of the
+   module's revision history.
+
+   A corollary to the text above is that the relationship between two
+   module or submodule revisions cannot be determined by comparing the
+   module or submodule revision date alone, and the revision history, or
    revision label, must also be taken into consideration.
 
```

Comment:
A corollary to the text above is that the ancestry (derived relationship) between two
   module or submodule revisions cannot be determined by comparing the
   module or submodule revision date or label alone - the revision history must be consulted.